### DISCUSSION TITLE
Taro Vue打包成react native报错

### DISCUSSION THREAD

**Discussion by liyouxia920** [NervJS/taro#9819] [Q & A]
Installing React-Native related packages:
react@^17.0.0 react-native@^0.64.0 @tarojs/taro-rn @tarojs/rn-runner
React-Native related packages have been installed successfully.

ATTEHNTION: Package.json has been modified automatically, please submit it by yourself.

  #####   ##   #####   ####     #####  ######   ##    ####  #####    #    #   ##   ##### # #    # ######
    #    #  #  #    # #    #    #    # #       #  #  #    #   #      ##   #  #  #    #   # #    # #
    #   #    # #    # #    #    #    # #####  #    # #        #      # #  # #    #   #   # #    # #####
    #   ###### #####  #    #    #####  #      ###### #        #      #  # # ######   #   # #    # #
    #   #    # #   #  #    #    #   #  #      #    # #    #   #      #   ## #    #   #   #  #  #  #
    #   #    # #    #  ####     #    # ###### #    #  ####    #      #    # #    #   #   #   ##   ######

React-Native Dev server is running on port: 8081

To reload the app press "r"
To open developer menu press "d"

                    Welcome to Metro!
              Fast - Scalable - Integrated

warn No apps connected. Sending "reload" to all React Native apps failed. Make sure your app is running in the simulator or on a phone connected via USB.
Reloading app...
warn No apps connected. Sending "reload" to all React Native apps failed. Make sure your app is running in the simulator or on a phone connected via USB.
Reloading app...

就是编译过程没问题，但无法找到app。
猜测报错原因：
1. win10 as环境变量配置有问题

**Comment by honlyHuang** [comment]
Taro Vue 不支持打包成 react native
